src/app/examples/form-field-example/form-field-example.component.scss

```diff
@@ -0,0 +1,15 @@
+@import "../../../kirby/scss/utils";
```

Comment:
Correct me if i am wrong here. 
But is the problem not, that our cookbook app is mixed in with our "libary" code for kirby. 
So if we enable non relative paths in the cookbook, we risk that none relative paths slips out in the kirby code. 

And that would break it for the users of kirby?